### ISSUE TITLE
Command pallette tip doesn't work. Needs array to fix

### ISSUE THREAD

**Issue by dragos240** [bitsper2nd/merge-monokai-theme#5]
Your [instructions for adding commands](https://github.com/bitsper2nd/merge-monokai-theme#quick-tip) does not work by default. It needs to be surrounded in square brackets to form an array.

**Comment by bitsper2nd** [comment]
Fair enough. I will update the README to include the [ ] for the change theme commands setting.
